spec/amber/cli/commands/pipelines/pipelines_spec.cr

```diff
@@ -22,6 +22,7 @@ module Amber::CLI
 
         web_default_plugs = %w(
           Amber::Pipe::PoweredByAmber
+          Citrine::Pipe::I18n
```

Comment:
Lets have this as `Citrine::I18n` since this is it's own shard that can be used independently it should not inherit from Amber

Reply:
Hey,
I've modified citrine-i18n to not depend on Amber now (it simply includes HTTP::Handler instead of subclassing Amber::Pipe::Base).

In terms of pipe name, I kept it named Citrine::Pipe::I18n for now because it would be hard to rename pipe class to Citrine::I18n (because Citrine::I18n is already a module).

Are you suggesting that I rename Citrine::Pipe::I18n to Citrine::I18n::Pipe, or something else?

Reply:
> Are you suggesting that I rename Citrine::Pipe::I18n to Citrine::I18n::Pipe, or something else?

I think `Citrine::Pipe::I18n` is fine now that I understand

Reply:
Yes, but your comment was on-point -- it makes no sense that citrine-i18n puts everything under Citrine::I18n, but then puts pipe in Citrine::Pipe::I18n.
Also, "Pipe" is Amber-specific term and you said that the shard should not be amber specific.

So I suggest that we rename the pipe to Citrine::I18n::Handler (or Citrine::I18n::Pipe if need be). Any thoughts from your side on that?

Reply:
I thought the exact same name :) `Citrine::I18n::Handler` go for it. I didnt want for you to make more changes.